examples/offline_inference/disaggregated_prefill.py

```diff
@@ -0,0 +1,80 @@
+import os
+import time
+from multiprocessing import Event, Process
+
+from vllm import LLM, SamplingParams
+from vllm.config import KVTransferConfig
+
+
+def run_prefill(prefill_done):
+    os.environ["CUDA_VISIBLE_DEVICES"] = "0"
```

Comment:
Please add comments to explain that we use GPU 0 for prefill and GPU1 for decode.